### PR TITLE
use install-cylc-components action

### PR DESCRIPTION
**Pull request by wxtim**
Use the centrally defined meta-release management thinggy for installing Cylc components from the correct branches.

### Checklist

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] This is a change to a github actions workflow which does not affect user experience, documentation or dependencies.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
Maybe rebase this onto the bugfix branch so we can apply it to both.

File: .github/workflows/test.yml
Comment (oliver-sanders) [inline .github/workflows/test.yml:1]: If renaming actions, this file needs to be updated - https://github.com/cylc/cylc-admin/blob/master/docs/status/branches.json

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -26,10 +26,20 @@ jobs:
...
 
-      - name: Install deps
-        run: |
-          pip install 'cylc-flow[empy] @ git+https://github.com/cylc/cylc-flow@master'
-          pip install 'metomi-rose @ git+https://github.com/metomi/rose@master'
+      # - name: Get Graphviz
+      #   run: |
+      #     conda install graphviz
+      #     conda install pygraphviz
+
```
Comment (oliver-sanders) [inline .github/workflows/test.yml:33]:
```suggestion
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -26,10 +26,20 @@ jobs:
...
+      - name: install libs
+        uses: cylc/release-actions/install-cylc-components@v1
+        with:
+          cylc_flow: true
+          cylc_flow_tag: ${{ inputs.cylc-flow-tag }}
+          cylc_flow_opts: 'empy, tests'
```
Comment (oliver-sanders) [inline .github/workflows/test.yml:39]: No need for the cylc-flow test dependencies.

```suggestion
          cylc_flow_opts: 'empy'
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -26,10 +26,20 @@ jobs:
...
+      # - name: Get Graphviz
+      #   run: |
+      #     conda install graphviz
+      #     conda install pygraphviz
+
+      - name: install libs
+        uses: cylc/release-actions/install-cylc-components@v1
+        with:
+          cylc_flow: true
+          cylc_flow_tag: ${{ inputs.cylc-flow-tag }}
```
Comment (oliver-sanders) [inline .github/workflows/test.yml:38]: This workflow doesn't have an input called `cylc-flow-tag` so this is doing nothing.

```suggestion
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -26,10 +26,20 @@ jobs:
...
+          cylc_flow_opts: 'empy, tests'
+          metomi_rose: true
+          metomi_rose_tag: ${{ inputs.metomi-rose-tag }}
```
Comment (oliver-sanders) [inline .github/workflows/test.yml:41]:
```suggestion
```

**Review by MetRonnie — APPROVED** [review]

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**